demo/assets/html/panoviewer_basic_example.html

```diff
@@ -0,0 +1,63 @@
+<!doctype html>
+<html lang="en">
+	<head>
+		<meta charset="utf-8">
+		<title>eg.view360.SpinViewer Basic Example</title>
+		<meta name="description" content="SpinViewer">
+		<meta name="author" content="egjs">
+		<meta name="viewport" content="width=device-width,initial-scale=1.0,minimum-scale=1.0,maximum-scale=1.0,user-scalable=no">
+		<link rel="stylesheet" href="../css/bootstrap.min.css">
+		<link rel="stylesheet" href="../css/docs.css">
+		<style>
+			#panoviewer-showcase {
+				position: relative;
+				width: 100%;
+				max-width: 400px;
+				height: 400px;
+				margin: 0 auto;
+			}
+		</style>
+	</head>
+	<body>
+		<div class="container">
+			<div class="bs-docs-section">
+				<h1 class="page-header"><a></a>PanoViewer</h1>
+				<p class="lead">
+					You can rotate by touch or motion (gyro).
+				</p>
+				<div class="bs-example">
+					<div id="panoviewer-showcase"></div>
+				</div>
+				<p>
+					For more code, you can use developer tools of browser.
+				</p>
+				<figure class="highlight">
+					<pre>
+						<code class="language-js" data-lang="js">
+var panoViewer = new eg.view360.PanoViewer(container, {
+	image: "../img/test_equi.png"
+});
+						</code>
+					</pre>
+				</figure>
+			</div>
+		</div>
+		
+		<script src="../dist/view360.pkgd.js" type="text/javascript"></script>
+		<script>
+			var container = document.getElementById("panoviewer-showcase");
+			var PanoViewer = eg.view360.PanoViewer;
+			var panoViewer = new PanoViewer(container, {
+				yaw: 180,
+				image: "../img/test_equi.png"
+			}).on("ready", function() {
+				panoViewer.lookAt({
+					yaw: 0
+				}, 2000);
+				// window.addEventListener("resize", function() {
+				// 	panoViewer.updateViewportDimensions();
+				// });
+			});
+		</script>
+	</body>
+</html>
```

Comment:
If you want to provide the simplest example first, how about removing the **ready** handler and make container have fixed size?